README.md

```diff
@@ -0,0 +1,69 @@
+# Client CRs with Quarkus distribution
+
+This repository contains examples with a temporary workaround for the missing Client CR in the [new Keycloak Operator](https://github.com/keycloak/keycloak/tree/main/operator).
+
+## Motivation
+
+To provide the best experience, the new Operator will use a new approach to manage Keycloak resources, such as Realms, Clients and Users. This approach will leverage the [new storage architecture](https://www.keycloak.org/2022/07/storage-map.html) and future immutability options, making the CRs the declarative single source of truth. In comparison to the [legacy Operator](https://github.com/keycloak/keycloak-operator), this will bring high robustness, reliability and predictability to the whole solution.
+
+However, since the new storage is still WIP and currently is lacking some features like the options for immutability that are crucial for the CRs, the new Operator is currently missing those CRs for managing Keycloak resources. The missing CRs will be added once the new store has support for it.
```

Comment:
Same comment as in https://github.com/keycloak/keycloak-web/pull/329#pullrequestreview-1088584195 applies here

Reply:
Good catch, thanks! Updated the PR.